test/smoketests/smoketest.test.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const chai = require('chai'),
+  expect = chai.expect,
+  sinon = require('sinon'),
+  Support = require('../support'),
+  DataTypes = require('sequelize/lib/data-types'),
+  current = Support.sequelize;
+
+describe(Support.getTestDialectTeaser('Model'), () => {
```

Comment:
You need to add your test in the integration test folder

Reply:
okay i will make the changes.